src/internal.c

```diff
@@ -1879,6 +1879,14 @@ int InitSSL_Ctx(WOLFSSL_CTX* ctx, WOLFSSL_METHOD* method, void* heap)
     ctx->noPskDheKe = 1;
 #endif
 
+#if defined(WOLFSSL_QT)
+    /* Qt retrieves supported cipher list at initialization 
+     * from get_cipher_compat().
+     * Qt doesn't not allow to use a cipher if it is not in the supported list.
```

Comment:
Let's re-word this to "Qt doesn't allow use of a cipher if it is not in the supported list."

Reply:
Thanks!